build.fsx

```diff
@@ -209,6 +209,19 @@ Target "DocFx" (fun _ ->
                     DocFxJson = docsPath @@ "docfx.json" })
 )
 
+//--------------------------------------------------------------------------------
+// Cleanup
+//--------------------------------------------------------------------------------
+
+FinalTarget "KillCreatedProcesses" (fun _ ->
```

Comment:
This needs to get called earlier inside `build.fsx`

Reply:
I followed the same setup on this build.fx

https://github.com/petabridge/Akka.Persistence.Extras/blob/73fb9c66b36452e1f83d93857ce4b1b7b16b00f2/build.fsx#L282-L294

I also did this on another project, should this be towards the beginning of the file?

Reply:
This is what you're missing: 

https://github.com/petabridge/Akka.Persistence.Extras/blob/73fb9c66b36452e1f83d93857ce4b1b7b16b00f2/build.fsx#L43

Reply:
Ahh I see, completely missed this. Fixing that now